microsoft/botframework-solutions
# [Botskills] Improve authentication process for verbosity

## Description
- Add logic to print the commands executed during the authentication process if `--verbose` is present.
- Improve the warning message when there's no `Azure Active Directory v2` authentication connection.

## Testing Steps
1. Go to `AI\lib\typescript\botskills\`.
1. Open a terminal in that location.
1. Run the command `npm install` to install dependencies.
1. Run the command `npm run build` to build the project.
1. Run the command `npm link` to symlink the package folder.
1. Run the command `botskills connect` with proper arguments, including `--verbose`.
1. You can check all the authentication commands executed will be printed.
